### PR TITLE
use cc to generate cshim

### PR DESCRIPTION
**Pull request by usamoi**
It should enable cross-compiling if `cshim` feature is enabled. It's tested on my machine and it works correctly.

### CODE REVIEW

**Comment by workingjubilee** [comment]
I am so glad I added that packaging check.

**Comment by workingjubilee** [comment]
It catches only the weirdest, silliest issues, but man, pgrx sure does have plenty of those.

File: pgrx-bindgen/Cargo.toml [hunk 1]
```diff
@@ -20,3 +20,4 @@ bindgen = { version = "0.70.1", default-features = false, features = ["experimen
 clang-sys = { version = "1", features = ["clang_6_0", "runtime"] }
 quote = "1.0.33"
 shlex = "1.3" # shell lexing, also used by many of our deps
+cc = "1.0"
```
Comment (workingjubilee) [inline pgrx-bindgen/Cargo.toml]: I know it doesn't matter.

compels me, tho'.
```suggestion
cc = "1.0"

```

**Review by workingjubilee — COMMENTED** [review]
Thoughts, @eeeebbbbrrrr?

**Comment by usamoi** [comment]
👀, @eeeebbbbrrrr ?

**Review by eeeebbbbrrrr — APPROVED** [review]

**Comment by eeeebbbbrrrr** [comment]
I’m sorry. I looked at this when it came in and then… who knows!

I like it. I’ll let @workingjubilee press the merge button, but this LGTM.

**Comment by eeeebbbbrrrr** [comment]
Oh. Does this add any new system dependencies,

I don’t guess it would but wanted to ask.

**Comment by workingjubilee** [comment]
It should effectively reduce them since we no longer need to know about pgxs.mk to build the cshim.

**Comment by eeeebbbbrrrr** [comment]
Hmm. Neat.

would be nice if we could drop the cshim entirely.

**Pull request merged**